core/src/modules/filterHelper.ts

```diff
@@ -90,10 +98,25 @@ export namespace FilterHelper {
     }
   }
 
+  /**
+   * Validate filters for a Property
+   *
+   */
   export async function validateFilters(
     instance: Property | Schedule,
     filters: FiltersWithKey[]
   ) {
+    const deprecatedFilters = {
+      equals: "eq",
+      "does not equal": "ne",
+      "greater than": "gt",
+      "greater than or equal to": "gte",
+      "less than": "lt",
+      "less than or equal to": "lte",
+      contains: "substring",
+      "does not contain": "notSubstring",
+    };
```

Comment:
I wonder if this check belongs here in the filterHelper, or in the configLoaders.  After the migration, the app shouldn't boot up with any of these old filters in place.  I think we would be more concerned with catching them at load time from a config file rather than holistically in the whole app (here)